Makefile

```
@@ -149,16 +143,46 @@ radixdnsalias-crd: temp-crds
 	cp $(CRD_TEMP_DIR)radix.equinor.com_radixdnsaliases.yaml $(CRD_CHART_DIR)radixdnsalias.yaml
 
 .PHONY: temp-crds
-temp-crds: controller-gen
-	echo "tempcrdrun"
-	${CONTROLLER_GEN} crd:crdVersions=v1 paths=./pkg/apis/radix/v1/ output:dir:=$(CRD_TEMP_DIR)
+temp-crds: bootstrap
+	controller-gen crd:crdVersions=v1 paths=./pkg/apis/radix/v1/ output:dir:=$(CRD_TEMP_DIR)
 
 .PHONY: delete-temp-crds
 delete-temp-crds:
 	rm -rf $(CRD_TEMP_DIR)
 
 .PHONY: lint
-lint:
+lint: bootstrap
 	golangci-lint run
 
+.PHONY: generate
+generate: bootstrap code-gen crds mocks
+
+.PHONY: verify-generate
+verify-generate: bootstrap generate
+	git diff --exit-code
+
+HAS_SWAGGER       := $(shell command -v swagger;)
+HAS_GOLANGCI_LINT := $(shell command -v golangci-lint;)
+HAS_MOCKGEN       := $(shell command -v mockgen;)
+HAS_CONTROLLER_GEN := $(shell command -v controller-gen)
```

```suggestion
HAS_CONTROLLER_GEN := $(shell command -v controller-gen;)
```
Missing semicolor. I got error `make: command: Command not found` when running for example `make generate`